raphtory/src/db/graph/edge.rs

```diff
@@ -163,20 +163,25 @@ impl<'graph, G: GraphViewOps<'graph>, GH: GraphViewOps<'graph>> BaseEdgeViewOps<
 
 impl<G: StaticGraphViewOps + InternalPropertyAdditionOps + InternalAdditionOps> EdgeView<G, G> {
     fn resolve_layer(&self, layer: Option<&str>, create: bool) -> Result<usize, GraphError> {
+        let valid_layers = self
```

Comment:
Same as above, if this is only in the error state, it should be calculated in the error state